libs/community/langchain_community/chat_models/mlflow.py

```diff
@@ -174,9 +174,16 @@ def _stream(
         )
         # TODO: check if `_client.predict_stream` is available.
         chunk_iter = self._client.predict_stream(endpoint=self.endpoint, inputs=data)
+        first_chunk_role = None
         for chunk in chunk_iter:
             choice = chunk["choices"][0]
-            chunk = ChatMlflow._convert_delta_to_message_chunk(choice["delta"])
+
+            chunk_delta = choice["delta"]
+            if first_chunk_role is None:
+                first_chunk_role = chunk_delta["role"]
```

Comment:
should we do this in case it's not present?
```suggestion
                first_chunk_role = chunk_delta.get("role")
```